Utilities/DataLoading/EdFi.LoadTools.Test/EdFi.LoadTools.Test.csproj

```diff
@@ -51,10 +51,10 @@
         <PackageReference Include="Microsoft.Owin.Testing" Version="4.0.0" />
         <PackageReference Include="Microsoft.Tpl.Dataflow" Version="4.5.24" />
         <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-        <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
-        <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
         <PackageReference Include="Moq" Version="4.8.2" />
         <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+        <PackageReference Include="NUnit" Version="3.12.0" />
```

Comment:
lets bump the packages for moq, nuint, nunite3testadapter and owin

![image](https://user-images.githubusercontent.com/288205/84931832-f9c24f00-b098-11ea-8ef1-1268edede6b2.png)

Reply:
I updated . Thanks !